vuepress/docs/next/docs/curate/bundle-version-types.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Bundle Classification *
+
+Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to Docker-based bundles, and this page clarifies the differences between the two, along with the improvements that were made in the new release.
+
+Some of the major improvements with the Docker-based bundles are:
+* Simplified bundle directory structure
+* Just one registry required with repositories created on the fly
+* Optimized file storage and transfer using a layered file-system, beneficial for large projects
+* Standardized bundle address format
+* Easier to modify, compose, and troubleshoot with decoupled processes 
+
+For specifications on Docker-based bundles, see the [Bundle Details](ecr-bundle-details.md) page. 
+### Bundle Comparison Table
+|Attribute** | [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](ecr-bundle-details.md)|
+| :- | :-------------------- | :---------------------  |
+|`descriptorVersion`| v1| v5
+| Bundle specifications |Define with descriptor files | Build with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by git tags |Bundle version set by Docker Tags. For MFEs, the version defaults to 1.0.0 unless set in [entando.json](ecr-bundle-details.md). 
+|Plugin Ingress| Generated by Entando or set in the CRD | Generated by Entando or customize in the [bundle descriptor](ecr-bundle-details.md#entando-bundle-conventions), entando.json 
+|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & Plugins have unique global IDs|
+|Initialize from Entando Hub|NA |[Direct initialization from Hub](../../tutorials/create/mfe/react.md)(switch to simple widget as available)  with `ent init BUNDLE-NAME –from-hub`|
+|Bundle Creation | `ent prj` commands| New modular `ent bundle` commands
+|Deploy & Install| `ent ecr deploy` and `ent ecr install` | No change 
+|Export Bundle|`ent bundler`|`ent generate-cr` and export files 
+|JHipster Integrated Blueprint Project| `ent jhipster blueprint` creates [MFE & MS template project](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)| NA
+|APIs| NA| [Manage APIs with `ent bundle` commands](../getting-started/ent-api.md). MFEs can make API claims to MSs within the namespace
+||| MFEs support service discovery by using the property apiClaim
+|Services |Utilizes Dockerfiles or CRDs to specify services. Keycloak is integrated with the ent CLI | Default services now include PostgreSQL, MySQL, and Keycloak. Additional services can be added with the [`ent bundle svc` commands](../getting-started/ent-svc.md).
```

Comment:
Should be Not Available for v1. Or maybe say "Custom Dockerfiles provided for JHipster Blueprint projects". It's not really integrated in v1.  Maybe also change the line to "Local Services" or "Development Services"? I'm trying to think how to distinguish between APIs and Services which look related and they are not here.